### PR TITLE
docs: Added two new pages for ISV integrations in the docs

### PR DESCRIPTION
**Pull request by madcommit**
### Reason for this change
This PR adds ISV partner pages for the docs website

### Description of changes

Added two files under /website/partners
### Description of how you validated changes
As this is just documentation, it doesn't change the code at all.

### Checklist

- [x] My code adheres to the [CONTRIBUTING GUIDE](https://github.com/awslabs/sbt-aws/blob/main/CONTRIBUTING.md)
- [x] I have updated the relevant documentation (if applicable).

---

*By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the project license.*

### CODE REVIEW

File: website/package.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
     "@docusaurus/preset-classic": "3.5.2",
     "@mdx-js/react": "^3.0.0",
     "clsx": "^2.0.0",
+    "docusaurus": "^1.14.7",
```
Comment (tobuck-aws) [inline website/package.json:22]: You don't need this dependency. I can remove it and push to this PR if you'd like. lmk.

File: website/package.json [hunk 2]
```diff
@@ -19,6 +19,7 @@
...
     "docusaurus-lunr-search": "^3.4.0",
```
Comment (tobuck-aws) [inline website/package.json:23]: Also, I think we can update this to `^3.5.0`. The lunr search isn't technically even hooked up yet (but will be at some point). We might even remove this until it is?

**Review by tobuck-aws — CHANGES_REQUESTED** [review]
Just a few comments. Lmk how you'd like to proceed.

File: website/package.json (hunk 2 above)
Comment (madcommit) [reply website/package.json]: Fixed the dependencies.

**Review by tobuck-aws — APPROVED** [review]
Thanks!

**Pull request merged**